src/drawing/__tests__/DrawingAnnotations-test.tsx

```diff
@@ -166,5 +212,60 @@ describe('DrawingAnnotations', () => {
 
             expect(wrapper.find(PopupDrawingToolbar).hasClass('ba-is-drawing')).toBe(true);
         });
+
+        describe('toolbar', () => {
+            const getWrapperWithToolbar = (
+                drawnPathGroups: Array<PathGroup>,
+                stashedPathGroups: Array<PathGroup>,
+            ): ReactWrapper =>
+                getWrapper({
+                    canShowPopupToolbar: true,
+                    isCreating: true,
+                    drawnPathGroups,
+                    stashedPathGroups,
+                });
+
+            test.each`
+                drawn         | stashed       | canComment
+                ${pathGroups} | ${[]}         | ${true}
+                ${[]}         | ${pathGroups} | ${false}
+                ${pathGroups} | ${pathGroups} | ${true}
+            `(
+                'should set canComment to $canComment if drawnPathGroups is $drawn and stashedPathGroups is $stashed',
```

Comment:
The use of `$drawn` here will dump a lot of JSON into the console. What are we trying to test?